native/Avalonia.Native/src/OSX/window.mm

```diff
@@ -2470,6 +2471,18 @@ virtual HRESULT Resize(double x, double y, AvnPlatformResizeReason reason) overr
                 [Window setFrameTopLeftPoint:ToNSPoint(ConvertPointY(lastPositionSet))];
             }
             
+            return S_OK;
+        }
+    }
+    virtual HRESULT SetTopMost (bool value) override
+    {
+        START_COM_CALL;
+        
+        @autoreleasepool
```

Comment:
I wonder if this is needed here

Reply:
if this function is called from COM (returns HRESULT so that seems to be the intention) then it needs both START_COM_CALL and autorelease pool.